### DISCUSSION TITLE
Release new Version & open Pulls

### DISCUSSION THREAD

**Discussion by jogibear9988** [jquery/esprima#2094] [General]
@ariya 
Is there a plan for a new version at npm?

and there are many open pull req. wich would move esprima forward, any plans to merge them?

**Comment by jginsburgn** [comment]
We need a release to be able to pull in the latest features of esprima, including more support for new ES syntax!

**Comment by yo-gi** [comment]
I'm curious what the timeline is for a new npm release! Seems like there are a lot of changes ready to go in the pipeline. Even a beta release would be helpful

**Comment by jginsburgn** [comment]
Hello? Is anybody here?

**Comment by jogibear9988** [comment]
@ljqx @ariya 

maybe you need people who help to maintain this project?

**Comment by ljqx** [comment]
> @ljqx @ariya
> 
> maybe you need people who help to maintain this project?

I don't have the permission to merge pulls or release new versions. Let's just wait for @ariya .

**Comment by jogibear9988** [comment]
There is also a fork from Contrast-Security-Inc (https://github.com/Contrast-Security-Inc/esprima)
wich seems to have many fixes...

https://github.com/jquery/esprima/compare/main...Contrast-Security-Inc:master

**Comment by jogibear9988** [comment]
@ljqx do you know if @ariya is still active here? 
maybe we should create an "official" fork, where we can continue development.

**Comment by jginsburgn** [comment]
+1 on the official (under the jQuery org?) fork.

**Comment by jogibear9988** [comment]
@jginsburgn 
thats the problem. seems no one of the org react to the messages here...

are here a few people who would like to help?
then I would create a fork (i will not really fork it, i will create a new repo with all commits. if you fork most people think it's not the main repo).
setup CI and add members.

or would somebody other do it?

if it's okay i would use my existing org: https://github.com/node-projects

**Comment by jogibear9988** [comment]
@loynoir what do you think?

**Comment by jogibear9988** [comment]
maybe we could ask some people of the jquery org if someone could get push access here?

@jeresig I think is the lead jQuery dev. Maybe He could help here?

**Comment by jogibear9988** [comment]
okay, if nothing happens here any more, I'll create a fork and setup CI & npm next week. I'll the try to merge most of the pulls.
If sometime development here speeds up again, we could remove it.
as a name? esprima-next?

**Comment by jginsburgn** [comment]
`esprima-next` SGTM.

**Comment by jogibear9988** [comment]
https://github.com/jquery/esprima/issues/2096

**Comment by kmatsumari** [comment]
Anyone still here? Last message was from 16 months ago, and the validator is still stuck on ES10 (ES11 has been out since 2020)

Any idea on if/when we could expect ES11 support?

**Reply by Meir017** [reply]
try https://github.com/node-projects/esprima-next
